LiskHQ/lisk
# Add husky and lint staged - Closes #1213

### What was the problem?
For automatic linting/formatting on commit.
### How did I fix it?
Integrated lint-staged on git pre-commit hook.
### How to test it?
Make changes to the file, and do `git add` and them when you do `git commit`, the `precommit` hook should automatically format and lint the code changes.
### Review checklist

* The PR solves #1213 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
